.travis.yml

```diff
@@ -18,8 +18,8 @@ env:
         # Add the BINSTAR_TOKEN environment variable. This has been generated with "" then "travis encrypt BINSTAR_TOKEN=<token_id>"
         - secure: "Z4k78LYglt90NV2lRN5kG3CBgCDqEMO/3i0l8NZWDWXNeXICdLx0FmXK3HBUwtJWSVIhgyX2LLsYBEOKQqyZYA/K9wtBdQp3gqo9CWxZwy3kP1yLSUeOzkcxebNY8oRjpclxgtp10VbgLpX6TxOucagI8K+hqEPIFUNsUEetF/ORhg73X7rjP3aBtrVmtzrmLPhwAOAOThG8kAbBHe/MPWXw8vBi9wxxS70upNACE0AwcPP1Mwec+3P7z5EmETlg+8TKi9qbwHwK3vxm42WiRrLQorHCwT7ejL5pgf2KagkgwqRJjE+EZO3cN+a14kd5QN0OPS6eoA79WWte449OFElN3op69n3R8wrwIlnW/QH83va0ArvlgI7bQ4+IRIzCLAKf2f3cQEaLUURYprxQFR7pnHNXvrxnXEVmhqGed2zdyZt6/oIEjobw95LOkcXdrWjdfjfsfeXEPDKA2VBx1ilbt9GJ4NjoY89Rl0luUuVyAMlqFA4Y9U99ObiCUHAO8nqnU6s9vOIskDBEAJNNkz82Wg81aJhwhKYBt+JDQTcfdOfoGpGfvwRr8a7eHi7owYt8r5g7zghK4xQkQa2blnoThDcUj7R91kfm2PiHUg3Q1c6BUvswQZxU4garc0Q90eXCuuP6dQicra0fcZNogFvbyh6pAUR7fd31YoEOv0k="
 
-        # Add the GH_TOKEN environment variable. (This is the "conda-forge.github.io" token for the conda-forge-manager user.)
-        - secure: "jTkMkCmdeSyrspb3aBl86xSgufeMMZwId5DOie/KgnjRKL+Qx1hskTp427RK/cvo1gvraBV2bulz6qkXjA7dYmouVbj9RhLa6MuTiUpOoPoP3fKdSeY9b7qsG4o/XiW2l0ED3Iwv5+ygcSEZ7QQjYnAUakzpplY9KE6t1YyLQ0TFojM4CLdhXSoCgsLcI6fDL4+3VrHWHJY8z/seCAxK+GJBs6ZrtqbQJXke5gTty2e1CBAy4zZw53w+Cj10QKMCbsLOuflXtT9TWyPdiK+whv9fNH7NCbfEELTYmm4f1yacdm11V//RzTsX3aU4QuKlEGrIoTN64ym0Kkqtcu63Pa69svpw2k06eV2w0sAFBaeK5/JVdwV/ZC7fnS79bPSDHwg8wqxT54F5fS2TN+H3XI1lMFBB3SiY/N0U+FsyfK7DIO0To5Q6QSNNDN+KxswRMyQitAW1kkGWvePDH2o0AXgfvYLK+09COU7xbSzvGo4Mrxft7mbpWdVJSw4j9wa03s9lrsVZmGges9oh/i1kiEtiEmNMPFFrmY0IXZdAIUi+rT8jOQGLrS7YbjhRQxJMtCbt9nULAm8zq7VYCA8IP6W43wzZVcY5qYn9f2n8JiBlNZT4yJbsOOcrwtMUiBVNnn9xjki7gqAsed7H4xX6FtcgHmuDI8JJ5OxsVugl+1Q="
+        # Add the GH_TOKEN environment variable. (This is the "staged-recipes" token for the conda-forge-manager user.)
+        secure: "gQ8MGFWnDxXNYKNcW9g8EaztbdmjE1n3MOIvBR9D3A3sOV+0PNb0VV7gS1NnYph+80MvWQBwZpIYC9aPxWt4uf7U1/0WZpgxrS6FRIdI85d3jwy7L3Qd94Luj1J5c0y3XAa3YhZYLx3TcMGuZYE6880vP+zkxo+jnYf5CUE6zxzttfDK/B8ps3zn+b/t0pBqrN5Xy3ZvotYGP4GSPmt2i5ZM3tnjOKTPwcBkHptq0k/nJWfrcOv8lhqea6TQDqHHtQsIqnL5pNBxMsE9mlonlMBr+fw9vImqiWl1bCzkAJ5vLi01eX6+ktpYinEx5E+SZbaRrYBIXifNONGFA87C4GgzzTG9ERHu7PRoFeZ3g9r+dd7yqggxgMtEylsJKWUOV4WtXmI97b+o/1T0+pk7luy1UgkzqXf/ArGLMeonb4gtwiBkKgbplW7sSkrZejoCkz16n2lCzLkEinCsMXOnfcmakW1cdsZJsBo9CCnA35uQuAbbb8jRBYSG3vuiwcYlXjVavrRsp7YbU5lB3Inj1bIDYNZKCbHZE5KxXiveRNwC8zGUMeetH2TW7qcb02r+ZfxdQQeTY+JwIMrbMQDgwCkFDVR2LvAJ/IrztRxEIM2GZAIalzSS5UJbWWMRxXcD4mWTXL8hJET+Tmolf/Mfk4T3Hy2+Zo7aFU08IZXa7iA="
```

Comment:
Looks like we lost the preceding `-` here. Is that expected?

Reply:
Good spot. Have updated.